src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -10565,7 +10565,7 @@ public enum NormalizationForm
         FormKC = 5,
         FormKD = 6,
     }
-    public readonly partial struct Rune : System.IComparable<System.Text.Rune>, System.IEquatable<System.Text.Rune>
+    public readonly partial struct Rune : System.IComparable, System.IComparable<System.Text.Rune>, System.IEquatable<System.Text.Rune>
```

Comment:
This seems to be missing the implementation of `CompareTo(Rune)`

Reply:
It's there, just a bit hidden. :)
The existing generic `CompareTo(Rune)` API is implicitly implemented so it's in alphabetical order with everything else.
The new non-generic `CompareTo(object)` API is explicitly implemented so it's at the end of the type declaration.

Reply:
Ah I see. I clicked on the expand lines in the UI but given the size of this file it didn't show it. Looking at it in raw form now I can see it 😄 

This is what GH was showing me:
![image](https://user-images.githubusercontent.com/22899328/87829359-ca3b6980-c833-11ea-9bbe-f9635f52a8a0.png)

🤦